### PR TITLE
Extension control changes

### PR DESCRIPTION
**Pull request by v-himkam**

### CODE REVIEW

File: src/AzSK.ADO/Framework/Abstracts/ServicesSecurityStatus.ps1 [hunk 1]
```diff
@@ -228,6 +228,18 @@ class ServicesSecurityStatus: ADOSVTCommandBase
...
 				
+				if([Organization]::InstalledextensionInfo)
+				{
+					$years = 2
+					if([Helpers]::CheckMember($svtObject.ControlSettings, "Organization.ExtensionsLastUpdatedInYears"))
+                    {
+                        $years = $svtObject.ControlSettings.Organization.ExtensionsLastUpdatedInYears
+					}
+					$folderpath=([WriteFolderPath]::GetInstance().FolderPath) + "\$($_.ResourceName)"+"_ExtensionInfo.csv";
+					[Organization]::InstalledextensionInfo | Select-Object extensionName,publisherId,KnownPublisher,publisherName,version,@{Name = "Too Old(>$($years)year(s))"; Expression = { $_.Updated } },lastPublished,@{Name = "Sensitive Permissions"; Expression = { $_.scopes} },@{Name = "NonProd (Galleryflags)"; Expression = { $_.ProductionReady}},@{Name = "NonProd (ExtensionName)"; Expression = { $_.Preview }},TopPublisher,PrivateVisibility | Export-Csv -Path $folderpath -NoTypeInformation
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Abstracts/ServicesSecurityStatus.ps1:239]: Add comments

Comment (v-himkam) [reply src/AzSK.ADO/Framework/Abstracts/ServicesSecurityStatus.ps1:242]: Added comment for fetching property from controlsettings

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1 [hunk 2]
```diff
@@ -4,6 +4,8 @@ class Organization: ADOSVTBase
     [PSObject] $ServiceEndPointsObj = $null
     [PSObject] $PipelineSettingsObj = $null
     [PSObject] $OrgPolicyObj = $null
+    static $InstalledExtensionInfo
+   ## $FolderPath=""
```
Comment (abhaydaga) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1:8]: remove this comment.

Comment (v-himkam) [reply src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1]: removed comment

**Review by abhaydaga — APPROVED** [review]

**Pull request merged**